app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -195,11 +195,30 @@ export const ViewContent: React.FC<IViewContentProps> = ({ setActiveContent }) =
       wo.userNotifications?.some((un) => un.userId === profile?.id),
   );
 
+  //Remove HTML tags, square brackets and line breaks before comparison.
+  const cleanString = (str: string | undefined) => str?.replace(/<[^>]*>?|\[|\]|\n/gm, '').trim();
+
+  //Difference ratio
+  const threshold = 0.1;
+
+  //Return true if difference between length of str1 & str2 is greater than 10%
+  const isDifferent = (str1: string | undefined, str2: string | undefined) => {
```

Comment:
When a function is not memoized `useCallback(...)` it can result in a lot of extra renders.  It's generally always best to extract a utility function if it doesn't require access to local values, or add it to a `useCallback(...)` if it does require local values.